frontend/lib/verification/verification_encoder.dart

```
@@ -0,0 +1,56 @@
+import 'dart:convert';
+
+import 'package:fixnum/fixnum.dart';
+
+import '../identification/base_card_details.dart';
+import '../identification/protobuf/card_verify_model.pb.dart';
+import 'verification_card_details.dart';
+
+String encodeVerificationCardDetails(
+    VerificationCardDetails verificationCardDetails) {
+  final cardDetails = verificationCardDetails.cardDetails;
+
+  final cardVerifyModel = CardVerifyModel();
+
+  cardVerifyModel.fullName = cardDetails.fullName;
+  cardVerifyModel.randomBytes = List<int>.from(cardDetails.randomBytes);
+  cardVerifyModel.expirationDate = Int64(cardDetails.unixExpirationDate);
+  switch (cardDetails.cardType) {
+    case "STANDARD":
+      cardVerifyModel.cardType = CardVerifyModel_CardType.STANDARD;
+      break;
+    case "GOLD":
+      cardVerifyModel.cardType = CardVerifyModel_CardType.GOLD;
+      break;
+    default:
+      cardVerifyModel.cardType = CardVerifyModel_CardType.STANDARD;
+      break;
+  }
```

Should we just throw an error in the `default` case?